R/page.R

```diff
@@ -73,6 +71,73 @@ validateCssPadding <- function(padding = NULL) {
   )
 }
 
+#' A sidebar page (i.e., dashboard)
+#'
+#' Create a dashboard layout with a full-bleed header (`title`) and [sidebar()].
+#'
+#' @inheritParams layout_sidebar
+#' @inheritParams page_fillable
+#' @param ... UI elements to display in the 'main' content area (i.e., next to
+#'   the `sidebar`). These arguments are passed to `layout_sidebar()`, which has
+#'   more details.
+#' @param title A string, number, or [htmltools::tag()] child to display as the
+#'   title (just above the `sidebar`).
+#'
+#' @export
+#' @seealso [layout_sidebar()] for 'floating' sidebar layouts.
+#' @seealso [accordion()] for grouping related input controls in the `sidebar`.
+#' @seealso [card()] for wrapping outputs in the 'main' content area.
+#' @seealso [value_box()] for highlighting values.
+#'
+#' @examplesIf interactive()
+#'
+#' library(shiny)
+#' library(ggplot2)
+#'
+#' ui <- page_sidebar(
+#'   title = "Example dashboard",
+#'   sidebar = sidebar(
+#'     varSelectInput("var", "Select variable", mtcars)
+#'   ),
+#'   card(
+#'     full_screen = TRUE,
+#'     card_header("My plot"),
+#'     plotOutput("p")
+#'   )
+#' )
+#'
+#' server <- function(input, output) {
+#'   output$p <- renderPlot({
+#'     ggplot(mtcars) + geom_histogram(aes(!!input$var))
+#'   })
+#' }
+#'
+#' shinyApp(ui, server)
+#'
+page_sidebar <- function(..., sidebar = sidebar(), title = NULL, fillable = TRUE, fill_mobile = FALSE, theme = bs_theme(), window_title = NA, lang = NULL) {
+
+  if (rlang::is_bare_character(title) || rlang::is_bare_numeric(title)) {
+    title <- h2(title, class = "bslib-page-title")
```

Comment:
Given its role on the page, I think this should be an `<h1>`

```suggestion
    title <- h1(title, class = "bslib-page-title")
```